### ISSUE TITLE
Change user's PWD to their home directory upon invocation of bash upon initial install

### ISSUE THREAD

**Issue by bbbbbrie** [Microsoft/WSL#1346]
This is an enhancement request. Upon initial install, the user's current working directory is:

````/mnt/c/Windows/System32````

It may be preferable to CD to the user's home directory to start.  

* Details
Once the initial installation of Bash on Windows is complete, bash is invoked on behalf of the user who is presented with a Bash shell. By default, Bash puts the user in their home directory upon invocation. It may be preferable for BashOnWindows to be consistent with this convention. Please note that if one invokes bash after the initial installation, the working directory is the home directory. This enhancement request is limited to the working directory after the initial installation.

* Windows build number: ````Version 1607 (OS Build 14393.21)````

* Steps / All commands required to reproduce the error from a brand new installation 
  1. Install Bash on Windows
  2. Observe present working directory ````pwd````

Thanks!

**Comment by aseering** [comment]
Hi @bbbbbrie , thanks for reporting this!  Interestingly, when I start WSL, it does put me in my home directory.

I've heard of the symptoms that you describe happening if something goes wrong with the installation.  Could that be the case for you?  Does this still happen if you uninstall and reinstall WSL?

**Comment by benhillis** [comment]
@bbbbbrie - Good suggestion. To describe what's happening, after install /bin/bash will launch from whatever folder you launched bash.exe from.  So I assume you initially ran bash.exe form C:\Windows\System32?

@aseering - bash.exe will start WSL in the home directory if you use the ~ argument.  I suspect you're launching through the shortcut which has that argument.

**Comment by FrostWalker** [comment]
@bbbbbrie This is the intended nature of running "bash.exe" directly since /mnt/c/Windows/System32 is the current working directory of the exe. If you run the shortcut "Bash on Ubuntu" it will correctly place you in the user home directory.

**Comment by rustyx** [comment]
Even if I set "**start in**" on the shortcut to **`%USERPROFILE%\AppData\Local\lxss\home\<my_user>`**, PWD becomes `/`.

I can't get bash to start in any subdirectory except `/` or `/mnt/...`

**Comment by therealkenc** [comment]
`bash.exe` is now depreciated. To start WSL in your home directory, use `wsl.exe ~`.

**Comment by Eugeny** [comment]
`~` doesn't work together with `-d` though:

```

C:\Users\x>wsl -d Debian ~

/bin/bash: /root: Is a directory
```

**Comment by therealkenc** [comment]
> ~ doesn't work together with -d though:
Use:
```
C:\>wsl ~ -d Debian
```
Ben needs to get an argument parsing library. Spawn a new issue if you feel motivated.

**Comment by rodrigo-x** [comment]
I have solved this with a simple: 

`**wsl ~**`

My wsl was opening in: "/mnt/c/WINDOWS/system32" then solved!
